src/validations/rules/sar.sch

```diff
@@ -175,6 +181,215 @@
+                                false()
+                    else
+                        true()"
+                unit:override-xspec="both">A finding must have an actor who is described in a SAP or SAR party assembly.</sch:assert>
+        </sch:rule>
+        <sch:rule
+            context="oscal:target">
+            <sch:let
+                name="implementation-status"
+                value="$fedramp-values/f:fedramp-values/f:value-set[@name = 'control-implementation-status']/f:allowed-values/f:enum/@value" />
+
+            <sch:assert
+                diagnostics="has-implementation-status-diagnostic"
+                doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Plans (SAR) §4.3"
+                fedramp:specific="true"
+                id="has-implementation-status"
+                role="error"
+                test="
+                    if (current()[@type = 'objective-id'])
+                    then
+                        oscal:prop[@ns = 'https://fedramp.gov/ns/oscal' and @name = 'implementation-status']
+                    else
+                        true()">A target of objective-id type must have a property of 'implementation-status'.</sch:assert>
+
+            <sch:assert
+                diagnostics="implementation-status-value-diagnostic"
+                doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Plans (SAR) §4.3"
+                fedramp:specific="true"
+                id="implementation-status-value"
+                role="error"
+                test="
+                    if (current()[@type = 'objective-id']/oscal:prop[@ns = 'https://fedramp.gov/ns/oscal' and @name = 'implementation-status'])
+                    then
+                        oscal:prop[@ns = 'https://fedramp.gov/ns/oscal' and @name = 'implementation-status']/@value[. = $implementation-status]
+                    else
+                        true()"
+                unit:override-xspec="both">The target's implementation-status property must be properly identified.</sch:assert>
+
+            <sch:let
+                name="satisfaction-status"
+                value="'satisfied', 'other-than-satisfied'" />
+            <sch:assert
+                diagnostics="has-satisfaction-status-diagnostic"
+                doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Plans (SAR) §4.3"
+                fedramp:specific="true"
+                id="has-satisfaction-status"
+                role="error"
+                test="
+                    if (current()[@type = 'objective-id'])
+                    then
+                        oscal:status
+                    else
+                        true()">A target of objective-id type must have a status.</sch:assert>
+
+            <sch:assert
+                diagnostics="satisfaction-status-value-diagnostic"
+                doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Plans (SAR) §4.3"
```

Comment:
The documentation reference should identify "_Guide to OSCAL-based FedRAMP Security Assessment **Results** (SAR)_".